File: Projects/rosalind/RNA.java
import java.io.*;
import java.util.*;

public class RNA {
	public static void main(String[] args) throws IOException {
		Scanner in = new Scanner(System.in);
		String seq = in.nextLine();
		System.out.print(seq.replaceAll("T", "U"));
		in.close();
	}
}

File: Projects/rosalind/RNA.kt
package rosalind

import java.io.IOException
import java.util.*

object RNA {
    @Throws(IOException::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val stdin = Scanner(System.`in`)
        val seq = stdin.nextLine()
        print(seq.replace("T".toRegex(), "U"))
        stdin.close()
    }
}